js/default-configs/layouts-config.js

```diff
@@ -341,6 +341,14 @@ window.flListLayoutConfig = {
         column: 'Category'
       }
     ],
-    'showSummaryFieldsInDetailView': true
+    'showSummaryFieldsInDetailView': true,
+    'detail-fields': [
```

Comment:
I'm not fully aware of how the property `showSummaryFieldsInDetailView` works, but if it does what it says, it seems to imply that all the fields in the`summary-fields` array will be shown in the detail view.

If that is the case, I can see that it already has an object for the image column:

```js
{
        id: 'gHlmJrlT',
        interfaceName: 'Image',
        location: 'Image',
        type: 'image',
        column: 'Image'
}
```

If that is the case, then why do we need to add a `detail-fields` array with just the Image field?

Reply:
@tonytlwu Code in summary fields affects fields that are located in "Summary view field" column
<img width="1440" alt="Summary field" src="https://user-images.githubusercontent.com/52824207/66553017-0928ea00-eb53-11e9-94af-746ed267ca6e.png">

Code in detail fields affects "Detail view field"
<img width="1440" alt="Detail fields" src="https://user-images.githubusercontent.com/52824207/66553115-34abd480-eb53-11e9-8da5-c3f8ffaac54d.png">